python/trifinger_simulation/gym_wrapper/envs/finger_push.py

```diff
@@ -63,23 +82,17 @@ def __init__(
             3,
         ]
 
+        # sets up the observation and action spaces for the environment,
+        # unscaled spaces have the custom bounds set up for each observation
+        # or action type, whereas all the values in the observation and action
+        # spaces lie between 1 and -1
         self.spaces = FingerSpaces(
             num_fingers=self.num_fingers,
             observations_keys=self.observations_keys,
             observations_sizes=self.observations_sizes,
             separate_goals=False,
         )
 
-        self.finger = SimFinger(
```

Comment:
Is this removed intentionally? Maybe I'm missing it but it doesn't seem to be created anywhere now.

Reply:
It is created in the first line of the __init__ method...

Reply:
Oh, yes... sorry, I somehow missed that.